bin/glodapBIO_to_hycom.py

```diff
@@ -36,12 +36,12 @@ def distance_on_unit_sphere(lat1, long1, lat2, long2):
   return arc
 
 searchradius = 20. # kilometers
-river_volume_flux_file   = open("/cluster/home/cagyum/DEVELOP/NERSC-HYCOM-CICE/input/rivers_ahype-ehype_clim_rev2.dat")
-river_nutrient_flux_file = open("/cluster/home/cagyum/DEVELOP/NERSC-HYCOM-CICE/input/globalNEWS_2012.dat")
+river_volume_flux_file   = open("/cluster/home/cagyum/DEVEL/NERSC-HYCOM-CICE/input/rivers_ahype-ehype_clim_rev2.dat")
+river_nutrient_flux_file = open("/cluster/home/cagyum/DEVEL/NERSC-HYCOM-CICE/input/globalNEWS_2012.dat")
```

Comment:
Is it not possible to pass the file path to the routine, so that each user does not have to edit the file path in order to use it?

Reply:
That script is not meant to be used as often, as the output is already in the input folder, unless the river loads change in future. I will look inti it.

Reply:
It's no big deal, we can leave it like this for now.